ocaml/util/xapi_version.ml

```diff
@@ -1,13 +1,13 @@
- let product_version () = Inventory.lookup ~default:"" "PRODUCT_VERSION" 
- let product_version_text () = Inventory.lookup ~default:"" "PRODUCT_VERSION_TEXT" 
- let product_version_text_short () = Inventory.lookup ~default:"" "PRODUCT_VERSION_TEXT_SHORT" 
- let platform_name () = Inventory.lookup ~default:"" "PLATFORM_NAME" 
- let platform_version () = Inventory.lookup ~default:"0.0.0" "PLATFORM_VERSION" 
- let product_brand () = Inventory.lookup ~default:"" "PRODUCT_BRAND" 
- let build_number () = Inventory.lookup ~default:"" "BUILD_NUMBER" 
+let product_version () = Inventory.lookup ~default:"" "PRODUCT_VERSION" 
+let product_version_text () = Inventory.lookup ~default:"" "PRODUCT_VERSION_TEXT" 
+let product_version_text_short () = Inventory.lookup ~default:"" "PRODUCT_VERSION_TEXT_SHORT" 
+let platform_name () = Inventory.lookup ~default:"" "PLATFORM_NAME" 
+let platform_version () = Inventory.lookup ~default:"0.0.0" "PLATFORM_VERSION" 
+let product_brand () = Inventory.lookup ~default:"" "PRODUCT_BRAND" 
+let build_number () = Inventory.lookup ~default:"" "BUILD_NUMBER" 
+let build_date () = Inventory.lookup ~default:"" "BUILD_DATE"
```

Comment:
Maybe we should use some default like "2017-01-01"? On a current build, I see no `BUILD_DATE` entry in `/etc/xensource-inventory` such that this would fall back to the default. How are we going to get this into the inventory or is there some other PR that takes care of this?

Reply:
Wrt to the default value, maybe we can keep the original default value `1970-01-01`.
There are some other PRs to other repos, I have posted them in the ticket.